test/cases/ostree-ng.sh

```diff
@@ -72,6 +72,7 @@ STAGE_REPO_ADDRESS=192.168.200.1
 STAGE_REPO_URL="http://${STAGE_REPO_ADDRESS}/repo/"
 QUAY_REPO_URL="docker://quay.io/osbuild/testing-rhel-edge-push"
 QUAY_REPO_TAG=$(tr -dc a-z0-9 < /dev/urandom | head -c 4 ; echo '')
+ARTIFACTS="ci-artifacts"
```

Comment:
let's maybe `mkdir` here so we don't have to do it twice? The same applies to the other script.